Update Redis exporter  to v1.48.0 (#3216)
grafana/agent: docs/sources/flow/reference/components/prometheus.exporter.redis.md

`ping_on_connect`             | `bool`         | Whether to ping the Redis instance after connecting. | | no
`incl_system_metrics`         | `bool`         | Whether to include system metrics (e.g. `redis_total_system_memory_bytes`). | | no
`incl_config_metrics`         | `bool`         | Whether to include Redis config values in metrics. | | no
`redact_config_metrics`       | `bool`         | Whether to redact sensitive elements from Redis config metrics, if `incl_config_metrics` is set. | `true` | no
`skip_tls_verification`       | `bool`         | Whether to to skip TLS verification. | | no

If `redis_password_file` is defined, it will take precedence over `redis_password`.

When `check_key_groups` is not set, no key groups are made.

The `check_key_groups_batch_size` argument name reflects key groups for backwards compatibility, but applies to both key and key groups.

The `script_path` argument may also be specified as a comma-separated string of paths, though it is encouraged to use `script_paths` when using
multiple Lua scripts.

Any leftover key groups beyond `max_distinct_key_groups` are aggregated in the 'overflow' bucket.

> The `is_cluster` argument should be used when connecting to a Redis cluster and using the `check_keys` and/or `check_single_keys` arguments.

Reviewer: Should? Or must?

```suggestion
The `is_cluster` argument must be set to `true` when connecting to a Redis cluster and using either of the `check_keys` and `check_single_keys` arguments.
```
